client/src/test/Components/form/AddressInfo.test.tsx

```
@@ -1,23 +1,31 @@
 // Mock the AddressForm component
-vi.mock('./AddressForm', () => ({
+vi.mock('../../../components/form/AddressForm', () => ({
   __esModule: true,
-  default: vi.fn(() => <div>Mocked AddressForm</div>),
+  default: vi.fn(({ address, onChange, title }) => (
+    <div>
+      <h2>{title}</h2>
+      <div data-testid={`${title}-district`}>{address.district}</div>
+      <div data-testid={`${title}-subdistrict`}>{address.subdistrict}</div>
+      <div data-testid={`${title}-village`}>{address.village}</div>
+      <button onClick={() => onChange({ district: 'Updated District', subdistrict: 'Updated Subdistrict', village: 'Updated Village' })}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)